metis-enrichment/metis-enrichment-rest/pom.xml

```diff
@@ -66,12 +66,7 @@
     </dependency>
     <dependency>
       <groupId>io.springfox</groupId>
-      <artifactId>springfox-swagger2</artifactId>
-      <version>${version.swagger}</version>
-    </dependency>
-    <dependency>
-      <groupId>io.springfox</groupId>
-      <artifactId>springfox-swagger-ui</artifactId>
+      <artifactId>springfox-boot-starter</artifactId>
```

Comment:
is this required, need to check with metis team on how the service is deployed

Reply:
I added this as a fix for swagger, as I was getting bean exception for swagger and the application was not starting. Now the application works fine expect the swagger mapping. For now swagger doesn't work anymore, there is a mapping dis ambiguity in this branch. But that should be fixed in a separate ticket.